ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -646,6 +646,32 @@ public async Task correctlyMaps_DenotificationValuesWhenNoDenotificationReason()
                 , notification.DenotificationDetails.OtherDescription);
         }
 
+        // Data for this has been taken from an edited test notification, relating to NTBS-2478
+        [Fact]
+        public async Task mapperDoesNotCloseNotificationIncorrectly()
```

Comment:
I think we need to update the name of this test to make it clearer what is actually being tested - it's really difficult for someone looking at this test with a fresh pair of eyes to work out what particular feature of the notification mapping is under test. (It's one of the reasons why I think holding the test data in csv files is a bad idea, but we seem to be stuck with that).